docs/FAQ.md

```diff
@@ -515,6 +515,29 @@ by `jj rebase` to rebase the changes in a commit. It's used in `jj log` to
 indicate which commits are empty. It's used in the `files()` revset function
 (and by `jj log <path>`) to find commits that modify a certain path. And so on.
 
+## How do I revert a merge commit? `jj revert <merge>` does nothing
+
+Since a merge commit is typically empty, because assuming two branches merge
+without conflict, there is no resolution that needs to be recorded in the
+merge commit itself, so its "empty". So `jj revert M` will not undo anything,
+because the merge itself was (presumpably) empty. Instead use
+`jj restore --from <first parent>` and add the respective message with
+`jj desc` or `jj commit`.
+
+Example:
+```text
+@
+|
+C
+| \
+B D
+|/
+A
+```
+To revert the merge in `C`, `jj restore --from B` and then describe the message
+with something like `jj desc -m "Revert merge <Cs commit-id>`.
```

Comment:
Either show what the graph would be after the command, or explain why the command reverts the merge.
Perhaps explain what `revert` actually means, if it's different from Git.

Reply:
I added some more words on what actually happens